theano/compile/debugmode.py

```diff
@@ -613,44 +613,10 @@ def _optcheck_fgraph(input_specs, output_specs, accept_inplace=False):
         instances already installed.
 
     """
-    orig_inputs = [spec.variable for spec in input_specs]
     updates = [spec.update for spec in input_specs if spec.update]
```

Comment:
I can't remove the ```orig_inputs``` because it's used here:

```python
fgraph = gof.fg.FunctionGraph(orig_inputs, orig_outputs,
                                  features=[equivalence_tracker])

```

do you want to remove the fgraph too?